### PR TITLE
English original and Russian translation links for header dropdown

### PR DESCRIPTION
**Pull request by gbezyuk**

### CODE REVIEW

File: themes/vue/layout/partials/language_dropdown.ejs [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <li class="nav-dropdown-container language">
   <a class="nav-link">言語</a><span class="arrow"></span>
   <ul class="nav-dropdown">
+    <li><a href="https://vuejs.org/" class="nav-link" target="_blank">English</a></li>
     <li><a href="https://vuejs.org/" class="nav-link" target="_blank">英語</a></li>
```
Comment (kazupon) [inline themes/vue/layout/partials/language_dropdown.ejs]: Can you remove this line please? 
this line is duplicate with the below line. 😉

**Review by kazupon — COMMENTED** [review]
Thank you for your PR!
I reviewed your PR. 👀 
Please check my review comment 💬

File: themes/vue/layout/partials/language_dropdown.ejs (hunk 1 above)
Comment (gbezyuk) [reply themes/vue/layout/partials/language_dropdown.ejs]: Oh, I see. Isn't it better to provide the link for English translation in English? I assume, one can be unable to read Japanese.

**Comment by gbezyuk** [comment]
I've added Japanese translations for foreign languages mentioned.

**Review by kazupon — APPROVED** [review]
Thanks! 👍

**Pull request merged**